docs/cog_guides/audio.rst

```diff
@@ -0,0 +1,3726 @@
+through Audio, that playlist will be fully cached locally and will not require any API credits to play songs.
+
+The following commands are relevant:
+
+* ``[p]genre`` - Lets users pick a Spotify music genre to queue music from.
+* ``[p]audioset countrycode`` - Lets guild owners specify what country code to prefer for Spotify searches, for the guild.
+* ``[p]audioset mycountrycode`` - Lets individual users pick what country code to prefer for Spotify searches of their own.
+
+.. _local-tracks:
+
+------------
+Local Tracks
+------------
+
+Audio can play music from a ``localtracks`` folder on the device where the bot is hosted. This feature is only available
+if your bot and your Lavalink.jar are on the same host, which is the default setup.
+
+To use this feature:
+
+1. Create a "localtracks" folder anywhere where Lavalink/your bot user has permissions to access, on the system.
+2. Use ``[p]audioset localpath <localtracks path>`` command to set the folder created above as local tracks folder.
+3. Create/move/copy/symlink your album folders (Subfolders containing your tracks) to the folder created in Step 1.
+4. Put any of Audio's supported files in the following folders:
+
+   * ``localtracks/<parent folder>/song.mp3``
+   * ``localtracks/<parent folder>/<child folder>/song.mp3``
+
+When using this localtracks feature, use ``[p]local`` commands. Use ``[p]play <parent folder>/song.mp3`` to play
+single songs. Use ``[p]local folder <parent folder>/<child folder>`` to play the entire folder.
+
+The following formats are supported:
+
+* MP3
+* FLAC
+* WAV
+* Matroska/WebM (AAC, Opus or Vorbis codecs)
+* MP4/M4A (AAC codec)
+* OGG streams (Opus, Vorbis and FLAC codecs)
+* AAC streams
+
+The following files are partially supported:
+
+* .ra
+* .wav
+* .opus
+* .wma
+* .ts
+* .au
+* .mov
+* .flv
+* .mkv
+* .wmv
+* .3gp
+* .m4v
+* .mk3d
+* .mka
+* .mks
+
+The following files are **NOT** supported:
+
+* .mid
+* .mka
+* .amr
+* .aiff
+* .ac3
+* .voc
+* .dsf
+* .vob
+* .mts
+* .avi
+* .mpg
+* .mpeg
+* .swf
+
+.. _dj-role-and-voteskip:
+
+--------------------
+DJ Role and Voteskip
+--------------------
+
+Audio has an internal permissions system for restrictions to audio commands while other people are listening to
+music with the bot. Bot owners, server admins and mods bypass these restrictions when they are in use.
+
+``[p]audioset dj`` will turn on the DJ role restriction system, ``[p]audioset role`` will let you choose or
+reassign the DJ role, and if you wish to make non-privileged users vote to skip songs, voteskip can be enabled
+with ``[p]audioset vote``.
+
+If a non-privileged user is listening to music alone in the channel, they can use commands without restrictions,
+even if DJ role or voteskip settings are active.
+
+.. _sound-quality-issues:
+
+--------------------
+Sound Quality Issues
+--------------------
+
+Laggy audio is most likely caused by:
+
+* A problem with the connection between the host machine of your bot to the Discord server.
```

Comment:
The machine hosting lavalink - as users can use unmanaged nodes on remote machines